pyDeltaRCM/init_tools.py

```diff
@@ -461,3 +461,27 @@ def init_subsidence(self):
             self.subsidence_mask[:self.L0, :] = False
 
             self.sigma = self.subsidence_mask * self.sigma_max * self.dt
+
+    def load_checkpoint(self):
+        """Load the checkpoint from the .npz file."""
+        ckp_file = os.path.join(self.prefix, 'checkpoint.npz')
+        checkpoint = np.load(ckp_file, allow_pickle=True)
```

Comment:
I guess this line will throw a `FileNotFound` error if there is no checkpoint created? I think that's probably fine, but is there any reason we would want to check for ourselves whether the file exists first and throw our own exception? Not sure, just thinking out loud.